apps/camera/js/controllers/controls.js

```diff
@@ -27,10 +27,32 @@ function ControlsController(app) {
   this.activity = app.activity;
   this.createView();
   this.bindEvents();
+  this.l10nGet = app.l10nGet;
+  this.updateSwitchLabels();
   debug('initialized');
 }
 
 /**
+ * Updates app `batteryStatus` and
+ * manages battery notifications.
+ *
+ * @private
+ */
+ControlsController.prototype.updateSwitchLabels = function () {
```

Comment:
looks like another copy paste style mistake `function ()` should be `function()`